Doc. Clarification about ArgMax/Min behavior (#50607)
ClickHouse/ClickHouse: docs/en/sql-reference/aggregate-functions/index.md

│ 2.3333333333333335 │                 1.4 │
└────────────────────┴─────────────────────┘
```

> Also you can use [Tuple](../data-types/tuple.md) to change NULL skipping behavior.

Reviewer: ```suggestion
Also you can use [Tuple](../data-types/tuple.md) to work around NULL skipping behavior. The a `Tuple` that contains only a `NULL` value is not `NULL`, so the aggregate functions won't skip that row because of that `NULL` value.
```

```sql

> select groupArray(b), groupArray(tuple(b)).1 from t_null_big;
> 
> ┌─groupArray(b)─┬─tupleElement(groupArray(tuple(b)), 1)─┐
> │ [2,2,3]       │ [2,NULL,2,3,NULL]                     │
> └───────────────┴───────────────────────────────────────┘

Reviewer: ```suggestion
SELECT
    groupArray(y),
    groupArray(tuple(y)).1
FROM t_null_big;

┌─groupArray(y)─┬─tupleElement(groupArray(tuple(y)), 1)─┐
│ [2,2,3]       │ [2,NULL,2,3,NULL]                     │
└───────────────┴───────────────────────────────────────┘
```

The motivation is to be consistent with the formatting and use the correct column names.

```

> 
> 
> 

Reviewer: ```suggestion
```

No need for extra empty lines.
